### DISCUSSION TITLE
Discord: Setting input field to type number

### DISCUSSION THREAD

**Discussion by Gluant101** [gluestack/gluestack-ui#1486] [Q&A]
TCI — Yesterday at 15:25
hey, how can i set gluestacks input field to type number? From what i can see it only takes text and password as types. Am i missing something

The5thElephant — Today at 00:38
I was also wondering about this

Link: https://discord.com/channels/1050761204852858900/1183708555312893952

**Comment by yann120** [comment]
I ended up finding a solution, you should add the prop inputMode with value equal to 'decimal' or 'numeric' (see documentation https://reactnative.dev/docs/textinput#inputmode)
This will change the opened keyboard on your phone
